### PR TITLE
create codeowners

### PR DESCRIPTION
**Pull request by joe-plumb**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by joe-plumb** [comment]
its fine! honest guv!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apis/poi @pmanlukas @MarkWme @matt-FFFFFF
+apis/trips @MarkWme @poornabavapalli
```
Comment (starlord-daniel) [inline .github/CODEOWNERS:2]:
```suggestion
apis/trips @MarkWme @poornabavapalli @pmanlukas
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apis/poi @pmanlukas @MarkWme @matt-FFFFFF
```
Comment (starlord-daniel) [inline .github/CODEOWNERS:1]:
```suggestion
apis/poi @pmanlukas @MarkWme @matt-FFFFFF @joe-plumb
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (joe-plumb) [reply .github/CODEOWNERS:2]: we could chat about it

File: .github/CODEOWNERS (hunk 2 above)
Comment (joe-plumb) [reply .github/CODEOWNERS:1]: wow yeah definitely much better

Comment (joe-plumb) [reply .github/CODEOWNERS:1]: thanks @starlord-daniel

**Comment by joe-plumb** [comment]
nice

**Comment by joe-plumb** [comment]
wow, great work

**Pull request merged**